modules/floating-buttons/assets/js/frontend/handlers/floating-buttons.js

```diff
@@ -274,6 +326,33 @@ export default class ContactButtonsHandler extends Base {
 	}
 
 	initDefaultState() {
+		// Manage accessibility
+		if ( this.elements.contentWrapper ) {
+			const randomishId = String(
+				Date.now().toString( 32 ) +
+					Math.random().toString( 16 ),
+			).replace( /\./g, '' );
+
+			const wrapperID = this.elements.contentWrapper.id
+				? this.elements.contentWrapper.id
+				: `e-contact-buttons__content-wrapper-${ randomishId }`;
```

Comment:
@louiswol94 to be fair this is something of a hangover from implementing this where the id didn't originally exist in the rendered markup & we had multiple variants being worked on so it was a catch for anything lost in the merge etc. We can remove this check or keep it to ensure accessibility is maintained should the rendered markup diverge.
@mserino any preference?

Reply:
Looking at the widget, I can't see the ID always applied. There is also the possibility for the user to change it in the Advanced tab, which makes it a little less secure because there could be duplicates if we leave the control to the user. Keeping it as it is I think we ensure that there are (virtually) no duplicates